source/backend/api/Models/Concepts/DispositionFile/DispositionFileTeamMap.cs

```diff
@@ -0,0 +1,33 @@
+using Mapster;
+using Entity = Pims.Dal.Entities;
+
+namespace Pims.Api.Models.Concepts
+{
+    public class DispositionFileTeamMap : IRegister
+    {
+        public void Register(TypeAdapterConfig config)
+        {
+            config.NewConfig<Entity.PimsDispositionFileTeam, DispositionFileTeamModel>()
+                .Map(dest => dest.Id, src => src.DispositionFileTeamId)
+                .Map(dest => dest.DispositionFileId, src => src.DispositionFileId)
+                .Map(dest => dest.Person, src => src.Person)
+                .Map(dest => dest.PersonId, src => src.PersonId)
+                .Map(dest => dest.Organization, src => src.Organization)
+                .Map(dest => dest.OrganizationId, src => src.OrganizationId)
+                .Map(dest => dest.PrimaryContact, src => src.PrimaryContact)
+                .Map(dest => dest.PrimaryContactId, src => src.PrimaryContactId)
+                .Map(dest => dest.TeamProfileType, src => src.DspFlTeamProfileTypeCodeNavigation)
+                .Map(dest => dest.TeamProfileTypeCode, src => src.DspFlTeamProfileTypeCode)
```

Comment:
TeamProfile and TeamProfileTypeCode seems to be duplicated (one is the string code, the other the typemodel representation of it). Perhaps the string one could be removed?